client/src/client.rs

```diff
@@ -34,7 +34,7 @@ use nakamoto_p2p::fsm;
 
 pub use nakamoto_net::event;
 pub use nakamoto_net::{Reactor, Waker};
-pub use nakamoto_p2p::fsm::{Command, CommandError, Hooks, Limits, Link, Peer};
+pub use nakamoto_p2p::fsm::{Command, CommandError, ConnDirection, Hooks, Limits, Peer};
```

Comment:
```suggestion
pub use nakamoto_p2p::fsm::{Command, CommandError, ConnDirection, Hooks, Limits, Peer};
```

I'm more in favor to keep `Link` here I found this more clear